packages/requisitions/src/ResponseRequisition/api/hooks.ts

```diff
@@ -1,7 +1,7 @@
-import { useQueryClient } from 'react-query';
 import { useMemo } from 'react';
 import {
-  useHostContext,
+  useAuthState,
```

Comment:
same as request - heavier use of `useResponseApi`

Reply:
sorry! changed to AuthContext and no doubt will give issues when merging 🙄

Reply:
no drama, easy one